policyengine_canada/variables/gov/cra/benefits/cdb/child_disability_benefit_eligible_children.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_canada.model_api import *
+
+
+class child_disability_benefit_eligible_children(Variable):
+    value_type = float
+    entity = Person
+    label = "Children eligible for Canada Child Disability Benefit"
+    unit = "Person"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        age = person("age", period)
+        disabled = person("is_dsiabled")
+        return disabled & (age < 18)
```

Comment:
Please make a parameter for age 18 - though I don't see this in the page. If the criterion is eligibility for DTC & CCB, let's use that instead.